shared/lib/pin_manager.py

```diff
@@ -216,6 +216,18 @@ def create_i2c(self, scl, sda, frequency=100000):
             (lambda: busio.I2C(scl, sda, frequency=frequency)),
         )
 
+    def create_uart(self, tx, rx, baudrate=50000):
+        """
+        Creates and returns ManagedDevice wrapping a busio.UART on the two specified pins
+        with the specified baudrate, or returns one from the cache if one has already
+        been created.
+        """
+        return self._create_general_device(
+            [tx, rx],
+            (busio.UART, baudrate),
```

Comment:
The `baudrate` parameter should not be used as a part of the key that identifies a `busio.UART` as a particular hardware device since the library allows `baudrate` to be reconfigure.